tailwind.config.js

```diff
@@ -7,6 +7,12 @@ module.exports = {
     './src/modules/**/*.{js,ts,jsx,tsx}',
     './src/layouts/**/*.{js,ts,jsx,tsx}',
   ],
+  safelist: [
+    {
+      pattern:
+        /(bg|text|border|ring|ring-offset)-scale-(good|medium|critical)(-dark)?/,
```

Comment:
Very schön 💅 ! Very sicher 🔒 !

Reply:
Yes exactly! Important since we generate some of these dynamically most of the time! With this refactoring I had noticed that a few classes were purged and so for instance the `TreeInfoHeader` would sometimes not have a color circle.